test/fluree/json_ld/impl/expand_test.cljc

```diff
@@ -641,6 +641,22 @@
            (expand/node {"@id" "foo"
                          "bar" {"@value" false}})))))
 
+(deftest empty-colon-test
+  (testing "Testing context key of ':' expands properly."
+    (is (= {"http://somedomain.org/age"     [{:idx   [":age"]
+                                              :type  nil
+                                              :value 33}]
+            "http://example.com/vocab/name" [{:idx   ["ex:name"]
+                                              :type  nil
+                                              :value "Frank"}]
+            :idx                            []}
+           (expand/node {"@context"
+                         {"ex" "http://example.com/vocab/"
+                          ":"  "http://somedomain.org/"}
```

Comment:
Does this have an edn equivalent?

Reply:
No, and I actually don't personally like it but the TURTLE spec uses it and I've come across many TTL examples that use it - so best to support.